azurerm/internal/services/cdn/cdn_profile_resource.go

```diff
@@ -66,10 +68,20 @@ func resourceArmCdnProfile() *schema.Resource {
 
 			"tags": tags.Schema(),
 		},
+
+		SchemaVersion: 1,
+		StateUpgraders: []schema.StateUpgrader{
+			{
+				Type:    resourceArmCdnProfile().CoreConfigSchema().ImpliedType(),
+				Upgrade: migration.CdnProfileV0ToV1,
+				Version: 0,
```

Comment:
actually this wants to be `1` since it's what this is migrating to

```suggestion
				Version: 1,
```

Reply:
As talked offline, this shall be kept.